addons/mass_mailing/models/res_config_settings.py

```diff
@@ -14,6 +14,8 @@ class ResConfigSettings(models.TransientModel):
     show_blacklist_buttons = fields.Boolean(string="Blacklist Option when Unsubscribing",
                                                  config_parameter='mass_mailing.show_blacklist_buttons',
                                                  help="""Allow the recipient to manage himself his state in the blacklist via the unsubscription page.""")
+    mass_mailing_reports = fields.Boolean(string="Mass Mailing Reports", config_parameter='mass_mailing.mass_mailing_reports',
+                                                 help="""Send mass mailing report to mass mailing responsible one day after sending a mass mailing.""", default=True)
```

Comment:
```suggestion
    mass_mailing_reports = fields.Boolean(string='Mass Mailing Reports', config_parameter='mass_mailing.mass_mailing_reports',
                                                 help='Send mass mailing report to mass mailing responsible one day after sending a mass mailing.', default=True)
```

Reply:
This file seems to have an inconsistent use of quotation symbols (`'`,`"` and `"""`). Should I change all of them to `'`?

Reply:
it's better to be consistent, at least on the same line I think (but it's just my "python style nazi opinion")